File: openscore-core/src/main/java/io/semantic/openscore/core/services/api/GruposService.java
package io.semantic.openscore.core.services.api;

import io.semantic.openscore.core.model.Grupo;

import javax.ws.rs.Consumes;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;
import javax.ws.rs.core.MediaType;

@Path("grupos")
@Produces(MediaType.APPLICATION_JSON)
@Consumes(MediaType.APPLICATION_JSON)
public interface GruposService extends GetService<Grupo> {
}

File: openscore-ui/services/GruposService.ts
import { CrearPartido } from "./../model/CrearPartido";
import { Grupo } from "./../model/Grupo";
import Service from "./Service";

export class GruposService extends Service<Grupo, CrearPartido, CrearPartido> {
  constructor() {
    super("grupos");
  }
}
